### PR TITLE
Add information to SFD dashboard page

### PR DESCRIPTION
**Pull request by allie1195abc**
the new dashboard was released so I added some notes about it!

Add a description of your changes with links to any relevant material.

### Date Needed (optional)

If the release date of a feature is known or estimated, provide it to give reviewers guidance on turnaround time.

### Reviewers

Possible roles follow. The PR submitter checks the boxes after each reviewer finishes and gives :+1:.

- [ ] Subject matter expert:
- [ ] Subject matter expert:
- [ ] Doc team review (sanity check, copy edit, or dev edit?): @edx/doc
- [ ] Product review:
- [ ] Partner support:
- [ ] PM review:

FYI: @sarina 

### Testing

- [ ] Ran ./run_tests.sh without warnings or errors

### HTML Version (optional)

- [ ] Build an RTD draft for your branch and add a link here

### Sandbox (optional)

- [ ] Point to or build a sandbox for the software change and add a link here

### Post-review

- [ ] Add a comment with the description of this change or link this PR to the next release notes task.
- [ ] Squash commits

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @allie1195abc!  Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.

**Comment by mphilbrick211** [comment]
Hi @allie1195abc! Thank you for your contribution! Please let us know if you have any questions on submitting your CLA form. Thanks!

**Comment by sarina** [comment]
Hi @allie1195abc - in the tutorial yesterday we asked you to submit a CLA, and I thought you did, but I confirmed with our legal team that we didn't get one. The form is at https://openedx.org/cla - if you can sign it, I can get this merged! Thanks for participating! 💯

File: en_us/shared/students/SFD_dashboard_profile.rst [hunk 1]
```diff
@@ -49,7 +49,8 @@ Change Course Information`.
 
 .. only:: Partners
 
-  Your dashboard has two pages.
+  * The dashboard has two tabs: 'Course Status' and 'Sort'
```
Comment (sarina) [inline en_us/shared/students/SFD_dashboard_profile.rst]: One other note, you'll need a blank line between this line and the next line.

You can follow instructions here to make changes to your PR: https://docs.openedx.org/en/latest/documentors/how-tos/make_changes_to_your_pull_request.html

**Comment by sarina** [comment]
@allie1195abc I love the updated changes. Could you sign the CLA at https://openedx.org/cla so we can get this merged?

**Comment by allie1195abc** [comment]
@sarina I thought I had signed the form but I just signed it again. Please let me know if you have any additional questions!

Thanks!
Allie

**Comment by sarina** [comment]
@allie1195abc hmm it looks like something's wrong on our side, unfortunately.

I can either ping you when the form is working again, or you can send me your full name and email address (to `sarina@axim.org`) and we'll send you the form directly.

**Comment by allie1195abc** [comment]
Hi @sarina ! No worries, here is my full name and email if you want to send me the form directly:

Thanks!
Allie

**Comment by sarina** [comment]
@allie1195abc UGH this is more complicated than I thought. I sent you an email, let's figure it out via email. Have a nice evening!

**Comment by openedx-webhooks** [comment]
@allie1195abc 🎉 Your pull request was merged!  Please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/edx-documentation&entry.867055334=https://github.com/openedx/edx-documentation/pull/2123&entry.1484655318=https://github.com/allie1195abc&entry.752974735=2023-03-28+18:58&entry.1917517419=2023-04-07+11:35&entry.2133058324=Yes) so we can improve your experience in the future.

**Pull request merged**